draft-ietf-scitt-architecture.md

```diff
@@ -304,10 +304,10 @@ Similarly, reputable Transparency Services are incentivized to secure their Appe
 
 The building blocks defined in SCITT are intended to support applications in any supply chain that produces or relies upon digital Artifacts, from the build and supply of software and IoT devices to advanced manufacturing and food supply.
 
-SCITT is a generalization of Certificate Transparency {{-CT}}, which can be interpreted as a transparency architecture for the supply chain of X.509 certificates.
+SCITT is a generalization of Certificate Transparency (CT) {{-CT}}, which can be interpreted as a transparency architecture for the supply chain of X.509 certificates.
 Considering CT in terms of SCITT:
 
-- CAs (Issuers) sign X.509 TBSCertificates (Artifacts) to produce X.509 certificates (Signed Statements)
+- CAs (Issuers) sign the ASN.1 DER encoded tbsCertificate stucture (Artifact) to produce an X.509 certificate (Signed Statements)
```

Comment:
Change stucture -> structure.

You also changed this from plural to singular and then you need to remove the s on Statements.  Though since this is a generic flow, I think you need to instead move it back to plural